Makefile

```diff
@@ -42,8 +42,8 @@ install-test:
 .PHONY: test
 test: install-test clean build
 	@chmod +x "./utils/tests.sh"; sync; \
-	export SECRET_PROJECT_ROOT="${PWD}"; \
-	export PATH="${PWD}/vendor/bats-core/bin:${PWD}:${PATH}"; \
+	export SECRET_PROJECT_ROOT="$(shell echo $${PWD})"; \
+	export PATH="$(shell echo $${PWD})/vendor/bats-core/bin:$(shell echo $${PWD}):$(shell echo $${PATH})"; \
```

Comment:
can we add a comment to explain why we're going through these gyrations with PWD?

Reply:
Added comments, see commit below

Reply:
Thank you!

Can we put this comment above the Makefile `.PHONY:test` line so it's not part of the test: command output?

Also maybe a comment like "the shell echo $${PWD}" construct is to get the Unix PATH env var on windows" would be clearer.